flutter_vlc_player/example/lib/single_tab.dart

```diff
@@ -181,15 +182,15 @@ class _SingleTabState extends State<SingleTab> {
                         hwAcc: HwAcc.FULL);
                     break;
                   case VideoType.file:
-                    Scaffold.of(context).showSnackBar(
+                    ScaffoldMessenger.of(context).showSnackBar(
```

Comment:
The `ScaffoldMessenger` breaking change hasn't been introduced into Flutter stable channel yet. Perhaps we should hold off this change until then?